install/check-minimum-requirements.sh

```diff
@@ -6,6 +6,12 @@ source "$(dirname $0)/_min-requirements.sh"
 function ver () { echo "$@" | awk -F. '{ printf("%d%03d%03d", $1,$2,$3); }'; }
 
 DOCKER_VERSION=$(docker version --format '{{.Server.Version}}')
+
+if [[ -z "$DOCKER_VERSION" ]]; then
+  echo "FAIL: Unable to get docker version, is the docker daemon running?"
+  exit 1
+fi
+
```

Comment:
Looks like a sidecar, yes? Or is this required for this PR?

Reply:
This is in response to the review by Amin here: https://github.com/getsentry/self-hosted/pull/1538#discussion_r912368681

I wanted to check the daemon was running as early as possible so later commands don't fail.